airflow/config_templates/default_airflow.cfg

```diff
@@ -445,8 +445,9 @@ access_control_allow_headers =
 # Specifies the method or methods allowed when accessing the resource.
 access_control_allow_methods =
 
-# Indicates whether the response can be shared with requesting code from the given origin.
-access_control_allow_origin =
+# Indicates whether the response can be shared with requesting code from the given origins. 
```

Comment:
```suggestion
# Indicates whether the response can be shared with requesting code from the given origins.
```

Installing the pre-commit hook and running these static tests yourself would have saved about 2 days 😁

Reply:
You're right ;) First time I contributed to project throw git ahah